en/docs/design/api-policies/attach-policy.md

```diff
@@ -1,19 +1,39 @@
 # Attaching Policies
```

Comment:
@ashera96 , try now to repeat the left side menu in the screenshots, because in instances when the left menu changes it will increase the maintenance overhead coz all the screenshots will have to be updated due to it having the left nav in it. Ideally, have it in one occurrence when telling them as to which menu that they need to click on and thereafter omit the left menu in the screenshot.